src/script-handlers/slotHandler.ts

```diff
@@ -9,18 +10,24 @@ export interface TypedParamTag extends ParamTag {
 // tslint:disable-next-line:no-var-requires
 import recast = require('recast')
 
-export default function eventHandler(documentation: Documentation, path: NodePath) {
+export default function slotHandler(documentation: Documentation, path: NodePath) {
   if (bt.isObjectExpression(path.node)) {
     const renderPath = path
       .get('properties')
-      .filter((p: NodePath) => bt.isObjectProperty(p.node) && p.node.key.name === 'render')
+      .filter(
+        (p: NodePath) =>
+          (bt.isObjectProperty(p.node) || bt.isObjectMethod(p.node)) &&
+          p.node.key.name === 'render',
+      )
 
     // if no prop return
     if (!renderPath.length) {
       return
     }
 
-    const renderValuePath = renderPath[0].get('value')
+    const renderValuePath = bt.isObjectProperty(renderPath[0].node)
+      ? renderPath[0].get('value')
+      : renderPath[0]
     recast.visit(renderValuePath, {
```

Comment:
Since you can return JSX from both methods and computed properties (and maybe even data?), I think we should visit the full Vue object here.

Reply:
It'll make the PR bigger though since we need to add all the relevant tests

Reply:
that is a good idea. In this case, you should create another handler so that not to polute the render function handler already present.